lib/mp4/transmuxer.js

```diff
@@ -1100,23 +1180,39 @@ Transmuxer = function(options) {
 
   // feed incoming data to the front of the parsing pipeline
   this.push = function(data) {
+    if (hasFlushed) {
```

Comment:
If we don't support dynamic reconfiguration of the transmuxer after flushing, most of this code wouldn't be necessary. What is the use-case for reconfiguring the transmuxer after flushes?